src/lib/es5.d.ts

```diff
@@ -1443,7 +1443,8 @@ interface ArrayConstructor {
     (arrayLength?: number): any[];
     <T>(arrayLength: number): T[];
     <T>(...items: T[]): T[];
-    isArray(arg: any): arg is any[];
+    isArray<T>(arg: ArrayLike<T>): arg is readonly T[];
+    isArray(arg: unknown): arg is any[];

```

Comment:
Shall it be `unknown` instead, since arg is `unknown`?
```suggestion
    isArray(arg: unknown): arg is unknown[];
```